hw_3/hw_3/ViewController.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  ViewController.swift
+//  hw_3
+//
+//  Created by Илья Мунгалов on 04.03.2022.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+             
+        /* Универсальные функции сложения, вычитания, умножения и деления (перегрузка функций, одинаковые названия для функций, разные типы параметров)*/
+        
+        sum(3, 4)
+        sum(3.4, 5)
+        sum(4, 3.5)
+        sum(23.43, 23.64)
+        
+        subtraction(5, 4)
+        subtraction(6.4, 5)
+        subtraction(6, 3.5)
+        subtraction(245.43, 23.64)
+        
+        multiply(3, 4)
+        multiply(3.4, 5)
+        multiply(4, 3.5)
+        multiply(23.43, 23.64)
+        
+       division(5, 4)
+       division(6.4, 5)
+       division(6, 3.5)
+       division(245.43, 23.64)
+        
+        // Вычислить сумму цифр четырехзначного числа
+        sumFourDigitNumberDigits(number: 9999)
+        
+        // Функция сравнения строк
+        compareStrings(firstString: "absfda", secondString: "absfda")
+        
+        // Циклический вызов функций - поломать приложение
+        //errorRecursion()
+        
+        // Функция возведения в степень с дефолтным параметром
+        powNumber(5)
+        
+        // Функция вычисления факториала числа (рекурсия)
+        factorial(10)
+        
```

Comment:
The same about this empty line. It doesn't split anything. We don't usually add empty lines after `{` and before `}`